docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,24 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The proposed solution is to have a Federated Catalog as own runnable service capable of crawling all the chosen catalogs and expose that data.
+
+## Rationale
+
+The Federated Catalog aims at providing the aggregation of catalogs from multiple participants in a centralized point to reduce latency.
+
+Choosing a solution that decouples it from the Control Plane (like the one used for the Data Plane) and able to be scalable will future-proof the Federated Catalog as a feature and embraces wider usage.
+
+## Approach
+
+The Federated Catalog running as own instance can be performed either by running respective Docker image in a container or directly running a generated `jar`. 
```

Comment:
no theoretical concepts needed. Please only highlight how it is going to be deployed in the Tractus-X EDC helm charts, i.e. as separate `deployment` template. Doing this will bring some increased complexity, for example we need an additional Kubernetes ingress definition, as opposed to: just an additional `path` in an existing ingress.

Here it would also be advisable to mention, that there is going to be an additional runtime (i.e. docker image), analogous to `edc-controlplane-XYZ` and `edc-dataplane-XYZ` and to list the required variants (in-mem, postgres persistence).

Reply:
I see, changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/48b1900fa819dbf0ec0f2652fc4fc07ef5cd8525).